packages/cli/generators/app/templates/test/ping.controller.test.ts.ejs

```diff
@@ -1,6 +1,6 @@
 import {createClientForHandler, supertest} from '@loopback/testlab';
 import {RestServer} from '@loopback/rest';
-import {<%= project.applicationName %>} from '../';
+import {<%= project.applicationName %>} from '../src/';
```

Comment:
Does this need to change?

Reply:
Yup, otherwise `tsconfig.json` would not behave correctly. The problem it was having before was that it would only build files in `src` and `test` because they were the only files specified to be compiled, but it also built `index.ts` when it wasn't supposed to because `ping.controller.test.ts` imported from it, meaning it **had** to be compiled in order for the test to work.

Although I guess now that we're adding back in `index.ts` in `includes` of `tsconfig.json`, it wouldn't matter so much. Do you think this change should be reverted?